src/qt/test/rpcnestedtests.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2016-2021 The Bitcoin Core developers
+// Copyright (c) 2016-2025 The Bitcoin Core developers
```

Comment:
so we don't have to update it every time...
```suggestion
// Copyright (c) 2016-present The Bitcoin Core developers
```